src/bindings/jna/plugins-kotlin/regex-example/src/README.md

```diff
@@ -0,0 +1,31 @@
+# Regex-Example Kotlin Plugin
+
+This plugin can be used to validate the values of keys via the `check/ktex-regex` metakey.
+
+It uses the value of the metakey as regex expression to check if newly added values match
+with the stored regex. If not, the plugin raises the `VALIDATION_SYNTACTIC` error code.
```

Comment:
Please use linebreaks with the end of the sentences, see documentation guidelines.

Please also support check/validation/message

Reply:
The breaks were removed with e1b9f82a1064a275398ccc73ebe297662e1c76a8 and the `check/validation/message` metakey is now supported since 59656e1893e0c751983f00f773ea72e162d9691c.